apps/web/src/server/views/applications/create/applicant/_website.njk

```diff
@@ -1,11 +1,20 @@
 {% extends "./applications-create-applicant-layout.njk" %}
 
-{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
 
 {% block stepTitle %}
 	What’s the applicant’s website?
 {% endblock %}
 
 {% block stepForm %}
-	work in progress
+	{{ govukInput({
+		id: "applicant.website",
+		name: "applicant.website",
+		classes: "govuk-!-width-one-third",
+		value: values['applicant.website'],
+		errorMessage: errors['applicant.website'] | errorMessage,
+		hint: {
+			text: "for example, https://www.heathrow.com"
```

Comment:
can you make a note that the correct text is this and the design needs updating please

Reply:
yep, note to design was already added to ticket.